docs/src/reference/configuration.md

```diff
@@ -319,22 +316,13 @@ Valid attributes for list entries in `inputs` are provided below.
 
 ### aligned
 * type: string
-* description: Path to a local or remote (S3, HTTP(S), GS) FASTA file with **aligned and _un_filtered** genome sequences. Sequences can be uncompressed or compressed.
+* description: Path to a local or remote (S3, HTTP(S), GS) FASTA file with **aligned** genome sequences. Sequences can be uncompressed or compressed.
```

Comment:
@huddlej do you want to add here that sequences can be unaligned _if_ one doesn't wish to use proximity subsampling, or do you feel this "feature" is not worth advertising as we may change the config in the future?

Reply:
@jameshadfield I think we should handle this better on the workflow side in a subsequent PR instead of advertising a workaround as a feature. Your idea from one of our previous discussions about having the workflow check whether priorities are needed sounds like the best way to handle this.